docs/src/pages/company/careers/designer.md

```diff
@@ -0,0 +1,101 @@
+# Designer
+
+<p class="description">Design is critical to the success of our mission. We are looking for skills that complement our Lead Designer. You will empower our audience that seeks to build outstanding-looking UIs with new tools.</p>
+
+## About the company
+
+MUI started as Material-UI, the most successful React implementation of Google’s Material Design.
+It has gotten a massive following not only due to the fidelity to Material but also because its number of components, carefully designed component API, obsession for details, and community engagement.
+Today, countless teams and organizations rely on our open-source projects to build their design system.
+
+A few years back, we started expanding our suite of products.
+We released [MUI X](/x/), a collection of advanced components, [MUI Design Kits](/design-kits/), the MUI Material components available in the most popular design tools out there, and [MUI Templates](/templates/), a set of pre-built UIs.
```

Comment:
This is not factual, MUI Design kits is meant to include all the React components, Joy, X, Material. 

```suggestion
We released [MUI X](/x/), a collection of advanced components, [MUI Design kits](/design-kits/), the MUI components available in the most popular design tools out there, and [MUI Templates](/templates/), a set of pre-built UI.
```